src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#"><img src="images/logo.png" alt="Moyo logo"></a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item is-active">
+            <a href="#apple" class="nav-link is-active">Apple</a>
+          </li>
+
+
+          <li class="nav-item">
+            <a href="#samsung" class="nav-link">Samsung</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#smartphones" class="nav-link">Smartphones</a>
+          </li>
+
+          <li class="nav-item" >
+            <a href="#laptops&computers"
+               class="nav-link testing"
```

Comment:
Redundant classes, you need to add color for all links when hovering, not only for this link
```suggestion
               class="nav__link"
```